### PR TITLE
Add echart method and `graphic` declaration.

### PR DESCRIPTION
**Pull request by AM-young-fun**
0. Add declaration of graphic.
1. Add method ```convertFromPixel```

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
    - https://ecomfe.github.io/echarts-doc/public/en/api.html#echartsInstance.convertFromPixel
    - https://ecomfe.github.io/echarts-doc/public/en/api.html#echarts.graphic

Tested and used in my production code.

### CODE REVIEW

**Comment by AntiMoron** [comment]
:+1

**Comment by AntiMoron** [comment]
👍

File: types/echarts/index.d.ts [hunk 1]
```diff
@@ -9,6 +9,8 @@ declare namespace ECharts {
         renderer?: string
     }):ECharts;
 
+    const graphic: any;
```
Comment (mhegazy) [inline types/echarts/index.d.ts]:
```ts
const graphic: {
    clipPointsByRect (points: number[][], rect: Rectangle) : number[][];
    clipRectByRect (targetRect: Rectangle, rect: Rectangle) : Rectangle;
}
```

**Review by mhegazy — CHANGES_REQUESTED** [review]
Also please add tests.

File: types/echarts/index.d.ts (hunk 1 above)
Comment (AntiMoron) [inline types/echarts/index.d.ts]: 👍 

FWIW, `LinearGradient` isn't included in the document but used in the official [demo](http://echarts.baidu.com/demo.html#bar-gradient).

```
import * as EChart from 'echarts';

new (<any>EChart.graphic).LinearGradient
```

Comment (AntiMoron) [inline types/echarts/index.d.ts]: type ```Rectangle``` needs to be added then...

Comment (AntiMoron) [inline types/echarts/index.d.ts]: Modified.

**Comment by AntiMoron** [comment]
@mhegazy Please review the code and decide whether it can merge now.

**Review by AM-young-fun — COMMENTED** [review]
Add typed object & apis.

**Pull request merged**